porebuilder/porebuilder.py

```diff
@@ -1,5 +1,8 @@
 import mbuild as mb
 import numpy as np
+from random import shuffle
+from copy import deepcopy
+from six import string_types
```

Comment:
`six` is not in the standard Python library, so we should add it to `requirements.txt`